content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -67,31 +71,65 @@ An important consideration in node-level logging is implementing log rotation,
 so that logs don't consume all available storage on the node. Kubernetes
 currently is not responsible for rotating logs, but rather a deployment tool
 should set up a solution to address that.
-For example, in Kubernetes clusters, deployed by the `kube-up.sh` script,
-there is a [`logrotate`](https://linux.die.net/man/8/logrotate)
-tool configured to run each hour. You can also set up a container runtime to
-rotate application's logs automatically, e.g. by using Docker's `log-opt`.
-In the `kube-up.sh` script, the latter approach is used for COS image on GCP,
-and the former approach is used in any other environment. In both cases, by
-default rotation is configured to take place when log file exceeds 10MB.
 
-As an example, you can find detailed information about how `kube-up.sh` sets
-up logging for COS image on GCP in the corresponding [script][cosConfigureHelper].
+### `logrotate`
 
-When you run [`kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) as in
-the basic logging example, the kubelet on the node handles the request and
-reads directly from the log file, returning the contents in the response.
+The [`logrotate`](https://linux.die.net/man/8/logrotate) tool can be configured
+on the host system to rotate logs periodically or when log files reach a
+certain size.
+
+Here's an example `logrotate` configuration file that can be placed in the
+`/etc/logrotate.d/` directory that would ensure no more than 5 log files of a
+maximum size of 5MB for each Pod on the Node.
+
+```
+/var/log/pods/*/*.log {
+    rotate 5
+    copytruncate
+    missingok
+    notifempty
+    compress
+    maxsize 5M
+    daily
+    dateext
+    dateformat -%Y%m%d-%s
+    create 0644 root root
+}
+```
+
+{{< note >}}
+By default, Kubernetes will store a Pod's logs in
+`/var/log/pods/{NAMESPACE_NAME_UID}/{POD}.log`. Kubernetes Services, in
```

Comment:
possible edit: By default, Kubernetes stores Pod log files in `/var/log/pods/{NAMESPACE_NAME_UID}/{POD}.log`, and Service log files in `/var/log/{SERVICE_NAME}.log`.